_src/scripts/scripts.js

```diff
@@ -371,6 +371,7 @@ async function loadLazy(doc) {
     loadHeader(doc.querySelector('header'));
   }
 
+  loadTrackers();
```

Comment:
asa merge iar performanta in rosu ? :)

Reply:
n-ar trebui ca nu e asteptata promisiunea. Ideea era sa descarce mai repede scriptul de tracking